test/config.sh

```diff
@@ -232,6 +233,10 @@ globalExcludeTests+=(
 	# https://github.com/docker-library/official-images/pull/1721#issuecomment-234128477
 	[clearlinux_no-hard-coded-passwords]=1
 
+	# alpine/slim openjdk images are non-headless and so can't do font stuff
```

Comment:
"alpine/slim openjdk images are non-headless" ?  you mean _are_ headless, right?